tests/test_api.rs

```diff
@@ -2938,3 +2938,37 @@ fn test_map_api() {
     );
   }
 }
+
+#[test]
+fn get_heap_statistics() {
+  let _setup_guard = setup();
+  let mut isolate = v8::Isolate::new(Default::default());
+    
+  let mut stats = v8::HeapStatistics::default();
+  isolate.get_heap_statistics(&mut stats);
+
+  assert!((stats.total_heap_size() >= std::usize::MIN));
```

Comment:
This is not a useful test because a `usize` is unsigned, which means that `foo >= 0` is always true...
Maybe you can make it more useful by checking that it has a reasonable value. It's going to be at least a few pages.